omod/src/main/java/org/openmrs/module/attachments/web/controller/AttachmentsController.java

```diff
@@ -35,6 +28,12 @@
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.multipart.MultipartHttpServletRequest;
 
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Iterator;
+
+import static org.openmrs.module.attachments.AttachmentsContext.getContentFamily;
+
 @Controller
 public class AttachmentsController {
```

Comment:
One last thing, can you [deprecate](https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html) this controller?

Reply:
is it ok to deprecate, even if it's still used by Client UI ?

Reply:
I mean the _real_ deprecation is getting rid of it. Setting the ad-hoc annotation just announces to devs that this API should be avoided as it is planned to go away.